### ISSUE TITLE
Batch normalization layer

### ISSUE THREAD

**Issue by cbovar** [cbovar/ConvNetSharp#13]

**Comment by lostmsu** [comment]
Just curious, with the current version, can I even do minibatches?

**Comment by lostmsu** [comment]
All right, from Classify2DDemo I guess I just need to set the last parameter of Volume's Shape to number of items in the minibatch. Can you confirm that?

**Comment by cbovar** [comment]
Yes you can do mini batch. And you are correct: the 4th dimension of the Shape is the batch size

**Comment by lostmsu** [comment]
@cbovar IMHO, you should mention that in README. Seems like an important detail.

**Comment by idow09** [comment]
Any plans on implementing this feature anytime soon?

**Comment by cbovar** [comment]
I'm not planning on implementing it any time soon. But PR are welcome of course :)
